torchgeo/datasets/benin_cashews.py

```diff
@@ -343,6 +345,7 @@ def _load_single_scene(self, date: str, bands: Tuple[str, ...]) -> Tensor:
             )
             with rasterio.open(filepath) as src:
                 self.tile_transform = src.transform
+                self.crs = src.crs
```

Comment:
Instead of setting these attributes, it might be safer to return these from the function.

Reply:
I agree, I went with the original style that was used for passing tile_transform to the mask function.

Do you want me to change the whole thing?

Reply:
Let's see if @calebrob6 has an opinion on this before changing anything, he wrote the original dataset class.

Reply:
returning `crs` and `tile_transform` would be less brittle / better overall, however this way wouldn't break anything

Reply:
done